src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import { TwitterShareButton, TwitterIcon } from "react-share";
+import * as analytics from "./GoogleAnalytics";
+
+const title = "Helpful Flashcards for Developers @ ";
+
+class Header extends Component {
+  onShare = () => {
+    analytics.logTwitterShare();
+  };
+
+  render() {
+    return (
+      <div className="container mt-3">
+        <div className="row">
+          <ul className=" header-right text-md-right">
+            <li className="list-inline-item">
+              <TwitterShareButton
+                url="http://nlaz.github.io/flashcards-for-developers/#/"
```

Comment:
Should we add a task for updating the domain name?

Reply:
think that she be done after the name is settled on